src/main/kotlin/lotto/constant/Rank.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.constant
+
+enum class Rank(val matchCount: Int, val prizeMoney: Int) {
+    FIRST(6, 2000000000),
```

Comment:
1

Reply:
```suggestion
    FIRST(6, 2_000_000_000),
```
언더 스코어를 활용하면 조금 더 가독성을 확보할 수 있습니다 :)